openslides/users/config_variables.py

```diff
@@ -0,0 +1,80 @@
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy
+
+from openslides.core.config import ConfigVariable
+
+
+def get_config_variables():
+    # Sorting
```

Comment:
Docstring: `Generator which yields all config variables of this app. They are grouped in 'Sorting' and 'PDF'. The generator has to be evaluated during app loading (see apps.py).`